src/libraries/Microsoft.Extensions.Hosting.Abstractions/src/IHostApplicationBuilder.cs

```diff
@@ -37,6 +38,11 @@ public interface IHostApplicationBuilder
     /// </summary>
     ILoggingBuilder Logging { get; }
 
+    /// <summary>
+    /// Allows enabling metrics and directing their output.
+    /// </summary>
+    IMetricsBuilder Metrics { get; }
```

Comment:
Doesn't this need to be a DIM to avoid creating a breaking change?

Reply:
This will be breaking change anyway for netstandard2 (I mean NETFX). having it as DIM is better to be less breaking. 

@Tratcher can you please ensure filing a breaking issue on the documentation repo?

Reply:
Luckily not, this interface is new in 8.
https://learn.microsoft.com/en-us/dotnet/api/microsoft.extensions.hosting.ihostapplicationbuilder?view=dotnet-plat-ext-8.0